charts/openshift-metering/templates/reporting-operator/reporting-operator-deployment.yaml

```diff
@@ -441,6 +445,11 @@ spec:
 {{- end }}
 {{- end }}{{/* end of authProxy.enabled */}}
       volumes:
+{{- if and .Values.networking.useGlobalProxyNetworking .Values.networking.proxy.config.trusted_ca_bundle }}
+      - name: proxy-user-ca-bundle
+        configMap:
+          name: user-ca-bundle
```

Comment:
We also don't need to mount the entire ConfigMap as we're only interested in the `data["ca-bundle.crt"]` item.

Reply:
We may also want to mark this as `readOnly: true` as there shouldn't be any write options needed.

Reply:
That is the only item in the user-ca-bundle